packages/core/src/components/header/header.scss

```diff
@@ -53,6 +55,16 @@ $dso-dropdown-link-color: $grijs-90;
   flex-wrap: wrap;
   padding-bottom: $u2;
   padding-top: $u2;
+
+  @media screen and (max-width: $screen-xs-max) {
+    max-width: 200px;
+  }
+
+   + .dropdown {
+     dso-dropdown-menu {
```

Comment:
Ik heb mijn bedenkingen of dit de juiste manier is. We stijlen nu elementen die onderdeel zijn van de API van Dropdown Menu, terwijl we we pure stijling niet als API strategie zien. Een gangbare API strategie voor Web Components zijn attributen, properties of CSS variablen.

@gofix hoe denk jij hierover?

Reply:
Ik snap helemaal wat je bedoelt. Eigenlijk zou je de dropdown menu en de header als onafhankelijke organismen willen beschouwen. Als die organismen invloed op elkaar zouden moeten uitoefenen, zou je dat inderdaad wellicht liever op een meer gecontroleerde manier doen. 

Wat mij betreft zijn CSS variabelen bij uitstek geschikt als we bv. eenvoudige eigenschappen als kleur, ruimte, font willen kunnen sturen vanuit een andere context, maar hier is veel meer sturing gewenst. 

Ik denk dat dat dan bijna een speciale (attribuut) modifier op `dso-dropdown-menu` rechtvaardigt (`in-header` of zo?). En dat dan alle styling voor `dso-dropdown-menu` die in deze `header.scss` terechtgekomen is, naar die modifier gaat luisteren?

Reply:
Ik weet het niet, voelt een beetje als over-engineering, en bovendien kiezen we er juist bij andere context-gevoelige styling (component A krijgt andere styling als in in context van component B 'anders' is) dat wel degelijk onder te brengen bij het component dat de context vormt (zie bv. `.dso-link-list` in footer.scss).

Reply:
Het voorbeeld wat jij aanhaalt is een HTML/CSS component, daar is de vrijheid beperkt tot styling en CSS variablen.

Het voelt mij ook als over-engineering. We laten het voor nu maar even.

Reply:
😗